plugins/explore/src/components/GroupsExplorerContent/GroupsDiagram.tsx

```diff
@@ -79,10 +88,19 @@ function RenderNode(props: DependencyGraphTypes.RenderNodeProps<any>) {
   }
 
   const ref = parseEntityRef(props.node.id);
+  const nameChunks = splitNameInChunks(props.node.name);
+  const objs = prepareForDisplay(nameChunks);
```

Comment:
this could have been one function call from the POV of this call site :)

maybe rename `objs` though? to something like `textLines` too?

Reply:
removed this part from the code